spec/models/course/assessment_spec.rb

```diff
@@ -144,10 +144,8 @@
     context 'after assessment was initialized' do
       subject { build(:assessment) }
 
-      it 'builds a folder' do
-        expect(subject.folder).to be_present
-
-        expect(subject).to be_valid
+      it 'sets the folder to have the same attributes as the assessment' do
+        subject.save
```

Comment:
because while it says after initialized, the implementation that was given actually only propagates the properties before save... lol https://github.com/Coursemology/coursemology2/blob/master/app/models/course/assessment.rb#L11